projects/plugins/jetpack/modules/enhanced-distribution.php

```diff
@@ -1,81 +1,10 @@
 <?php
 /**
- * Module Name: Enhanced Distribution
- * Module Description: Increase reach and traffic.
- * Sort Order: 5
- * First Introduced: 1.2
- * Requires Connection: Yes
- * Auto Activate: Public
- * Module Tags: Writing
- * Feature: Engagement
- * Additional Search Queries: google, seo, firehose, search, broadcast, broadcasting, creator
+ * Module formerly known as Enhanced Distribution.
+ *
+ * @deprecated 13.2.0
  *
  * @package automattic/jetpack
  */
 
```

Comment:
I think I would add a deprecation warning just in case that file is directly called somewhere for some reason:

```suggestion

_deprecated_file( basename( __FILE__ ), 'jetpack-$$next-version$$' );
```